docs/feature_unicode.md

```diff
@@ -65,6 +65,8 @@ The following input modes are available:
   To enable, go to _System Preferences > Keyboard > Input Sources_, add _Unicode Hex Input_ to the list (it's under _Other_), then activate it from the input dropdown in the Menu Bar.
   By default, this mode uses the left Option key (`KC_LALT`), but this can be changed by defining [`UNICODE_OSX_KEY`](#input-key-configuration) with another keycode.
 
+  This disables some Option based shortcuts including Option + Left Arrow and Option + Right Arrow (`moveWordLeftAndModifySelection` and `moveWordRightAndModifySelection`)
```

Comment:
I think this would be better here:
```suggestion
  **Note:** Using the Option (`KC_LALT`) key may disable some Option based shortcuts including Option + Left Arrow (`moveWordLeftAndModifySelection`) and Option + Right Arrow  (`moveWordRightAndModifySelection`).  You may want to use a different key if this is a problem.
```

Reply:
I'm struggling to find a resource that describes changing this key... In fact, if I could change it, I think I would so my beloved 'flip table' key could go back to working.

Did you find a resource that describes the process?

Reply:
In the code:
https://docs.qmk.fm/#/feature_unicode?id=input-key-configuration

On the system: https://docs.qmk.fm/#/feature_unicode?id=input-modes

>To enable, go to System Preferences > Keyboard > Input Sources, add Unicode Hex Input to the list (it's under Other), then activate it from the input dropdown in the Menu Bar.

Reply:
Oh I see. But I'm not sure that this would alter the behavior this line warns of; the OSX shortcuts  themselves are disabled. So unless you can tell OSX to listen to a different key (say Hyper or Meh keys), I think the Option Arrow keybinds will be broken.

Reply:
Well, I don't have a Mac, so I can't really comment on the settings and how exactly it works.  
My understanding is that you can change what key is used for Unicode input.  Which ... should fix this, in theory.  But beyond theory, I can't really help. 

However, I still do thing that the changes to the note/warning above are a good idea.  At the very least, it makes it easier to see.